on-prem/installation/installation_guide.md

```diff
@@ -4,28 +4,19 @@
 
 Stoplight consists of three container services:
 
-- The frontend app
-- The backend API
+- Stoplight Frontend
+- Stoplight Backend
 - A PostgreSQL database
 
-## Deployment environments
```

Comment:
I just removed because I didn't want to confuse the customer we were providing it to. They were only looking to install in OpenShift.

When we eventually release this publicly we'll want everything

Reply:
Not sure why this needs to be removed? It's clear the guide currently covers OpenShift, but we are supporting Unix at some point, right? And eventually I imagine this guide will grow to include that.